internal/v1/handler_blueprints.go

```diff
@@ -132,6 +132,40 @@ func (h *Handlers) GetBlueprint(ctx echo.Context, id openapi_types.UUID) error {
 	return ctx.JSON(http.StatusOK, blueprintResponse)
 }
 
+func (h *Handlers) GetImageTypesFromBlueprint(ctx echo.Context, id openapi_types.UUID) error {
+	userID, err := h.server.getIdentity(ctx)
+	if err != nil {
+		return err
+	}
+
+	ctx.Logger().Infof("Fetching blueprint %s", id)
+	blueprintEntry, err := h.server.db.GetBlueprint(ctx.Request().Context(), id, userID.OrgID())
+	if err != nil {
+		return err
+	}
+
+	blueprint, err := BlueprintFromEntry(blueprintEntry)
+	if err != nil {
+		return err
+	}
+
+	blueprintResponse := BlueprintResponse{
+		Id:             id,
+		Name:           blueprintEntry.Name,
+		Description:    blueprintEntry.Description,
+		ImageRequests:  blueprint.ImageRequests,
+		Distribution:   blueprint.Distribution,
+		Customizations: blueprint.Customizations,
+	}
+
+	var imageTypes []ImageTypes
+	for _, req := range blueprintResponse.ImageRequests {
+		imageTypes = append(imageTypes, req.ImageType)
+	}
+
+	return ctx.JSON(http.StatusOK, imageTypes)
```

Comment:
Why do you create new `BlueprintResponse` only to use `blueprintResponse.ImageRequests`? This can be simplified.